jest.config.json

```diff
@@ -1,6 +1,6 @@
 {
   "transform": {
-    "^.+\\.ts?$": "babel-jest"
+    "^.+\\.ts?x?$": "babel-jest"
```

Comment:
We can revert this setting if we convert the test file back to a `.ts` file

Reply:
Reverted